docs/nddata/subclassing.rst

```diff
@@ -275,8 +275,8 @@ To change the default value of an existing parameter for arithmetic methods::
 
     >>> ndd1 = NDDDiffAritDefaults(1, mask=False)
     >>> ndd2 = NDDDiffAritDefaults(1, mask=True)
-    >>> ndd1.add(ndd2).mask is None  # it will be None
-    True
+    >>> # No mask handling logic will return no mask:
+    >>> ndd1.add(ndd2).mask
```

Comment:
This is a small change to the edge case when no mask handling is defined. If the user doesn't specify how to handle masks, I'd argue that `NDDDiffAritDefaults` should not return a `NDDDiffAritDefaults` with a mask, since it is not clear what behavior the user could expect.